cmd/spec/main.go

```diff
@@ -1,23 +1,51 @@
 package main
 
 import (
-	"bytes"
 	"encoding/json"
-	"io/ioutil"
+	"os"
 	"reflect"
 
 	"github.com/RHEnVision/provisioning-backend/internal/payloads"
 	"github.com/getkin/kin-openapi/openapi3"
 	"github.com/getkin/kin-openapi/openapi3gen"
-	"github.com/ghodss/yaml"
+	"gopkg.in/yaml.v2"
```

Comment:
I wanted to move to `gopkg.in/yaml.v3` that we already use, but that reformats the whole yaml spec, which makes it quite hard to review.